addons/account/models/account_move_line.py

```diff
@@ -494,27 +494,29 @@ def _compute_account_id(self):
                            'account.move' AS model,
                            line.move_id AS id,
                            NULL AS account_type,
+                           NULL::integer AS company_id,
```

Comment:
Why do we need to cast NULL?

Reply:
![image](https://github.com/odoo/odoo/assets/47426643/cfa7d9cb-ee29-4787-8b4e-4ba4e49bf172)

With the Union ALL, each column must have the same type between queries, NULL is taken as string I guess